libparsec/crates/client/src/invite/claimer.rs

```diff
@@ -1845,6 +1851,8 @@ impl ShamirRecoveryClaimFinalizeCtx {
         .map_err(|e| anyhow::anyhow!("Error while saving the device file: {e}"))?;
 
         // After the device has been successfully saved, we still have to mark the invitation as completed.
+        // as it cannot be done by the greeter, as they cannot know if the process is complete
```

Comment:
```suggestion
        // because the greeter cannot know if the process is complete
```
Also, note the previous line ends in a ".", it should be removed